tests/test_datastructures.py

```diff
@@ -339,7 +337,7 @@ def test_multidict():
     assert repr(q) == "MultiDict([('a', '123'), ('b', '456')])"
 
     q = MultiDict([("a", "123"), ("b", "456")])
-    q.update(None)
+    q.update(None)  # type: ignore
```

Comment:
Not really sure what to do here. It was mentioned on the issue we might be able to remove this?

Reply:
From https://github.com/encode/starlette/pull/345 it seems because `.update()` is allowed (see the default ` = None`). I'm actually surprised passing an explicit `None` value is a problem for mypy. But I _think_ the intended test case here was `q.update()`. So, maybe...?

```suggestion
    q.update()
```

Reply:
Are you sure it's default `None`? I don't think so.
https://github.com/encode/starlette/blob/c8b9581e371968b4da42d0f2de01e4028d71ac3c/starlette/datastructures.py#L357-L365

Reply:
Oh that must have changed since #345 then (which _did_ default to `None`, hence `.update(None)` would have worked fine back then).

Reply:
But yeah, I think the suggestion looks good.

Reply:
Looks like the change happened in https://github.com/encode/starlette/pull/355/files#diff-e939ae940ce35a0db1544bfea4ab853be43ba89ea33e3cac713aa0b19f4bb875L229-L233 which did not update the tests;

I think we can remove this little test case altogether, then?

Reply:
Agreed.